packages/android_alarm_manager_plus/example/android/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.5.20'
```

Comment:
Are there any specific reason to rollback Kotlin version? If there are no issues with version which was used before please revert this change.